lib/ops/opsservice/operationgroup.go

```diff
@@ -257,17 +257,30 @@ func (g *operationGroup) onSiteOperationComplete(key ops.SiteOperationKey) error
 		return nil
 	}
 
-	site, err := g.operator.openSite(g.siteKey)
+	cluster, err := g.operator.openSite(g.siteKey)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 
+	if operation.IsCompleted() {
+		token, err := cluster.users().GetOperationProvisioningToken(key.SiteDomain, key.OperationID)
```

Comment:
Nit: I'd move this block into a separate method (`cleanOperationToken`) and maybe do it on a best-effort basis i.e. do not fail if this fails for some reason?